### DISCUSSION TITLE
Add like songs feature

### DISCUSSION THREAD

**Discussion by circles-00** [AzuraCast/AzuraCast#4455] [Ideas]
It would be nice if listeners could *optionally log in and save current playing song to a favorites list, or a playlist idk, so they can look it up later on youtube or spotify.

That'd be nice I think.

Anyways, thank you too all the developers once again for this amazing project.

**Comment by ivellios1988** [comment]
I believe it would be a better idea to implement such a feature on your website and grab song titles using AzuraCast's API.

**Reply by circles-00** [reply]
Thank you for that,  I will think about it and try to implement it.
